solutions/swb-reference/integration-tests/support/complex/paabHelper.ts

```diff
@@ -24,14 +24,20 @@ export class PaabHelper {
   private _setup: Setup;
   private _randomTextGenerator: RandomTextGenerator;
   private _outputError: boolean | undefined;
+  private _numberOfProjects: number = 3;
 
-  public constructor(outputError?: boolean) {
+  public constructor(numberOfProjects?: number, outputError?: boolean) {
```

Comment:
Does `numberOfProjects` need to be passed into the constructor? We can keep the scope smaller by passing in `numberOfProjects` to `createResources`. That's the only method using it. That way we won't need a class variable for a variable that's only being used in one method.

Reply:
Yeah, I realized it would have been better on createResources after I had refactored all the tests; I can go back through and update them all if we think it's worth changing, but I don't see a reason to do that now until we come back to make this into a proper singleton.